resources/public/api/conf/2.0/foreign_property_period_summary_create.yaml

```diff
@@ -0,0 +1,162 @@
+    <table>
+        <thead>
+            <tr>
+                <th>Header Value (Gov-Test-Scenario)</th>
+                <th>Scenario</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td><p>N/A - DEFAULT</p></td>
+                <td><p>Simulate success response.</p></td>
+            </tr>
+            <tr>
+                <td><p>NOT_FOUND</p></td>
+                <td><p>Simulates the scenario where the given resource could not be found.</p></td>
+            </tr>
+            <tr>
+                <td><p>OVERLAPPING</p></td>
+                <td><p>Simulates the scenario where the period summary overlaps with an existing period summary.</p></td>
+            </tr>
+            <tr>
+                <td><p>MISALIGNED</p></td>
+                <td><p>Simulates the scenario where the period summary isn't within the accounting period.</p></td>
+            </tr>
+            <tr>
+                <td><p>NOT_CONTIGUOUS</p></td>
+                <td><p>Simulates the scenario where the period summaries are not contiguous.</p></td>
+            </tr>
+            <tr>
+                <td><p>DUPLICATE_SUBMISSION</p></td>
+                <td><p>Simulates the scenario where a summary has already been submitted for the specified period.</p></td>
+            </tr>
+            <tr>
+                <td><p>TYPE_OF_BUSINESS_INCORRECT</p></td>
+                <td><p>Simulates the scenario where the supplied Business ID does not represent a foreign property business.</p></td>
+            </tr>
+        </tbody>
+    </table>
+
+
+  tags:
+    - Foreign Property Income & Expenses Period Summary
+
+  parameters:
+    - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+    - $ref: './common/pathParameters.yaml#/components/parameters/businessId'
+    - $ref: './common/pathParameters.yaml#/components/parameters/taxYear'
+    - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
+    - $ref: './common/headers.yaml#/components/parameters/contentTypeJson'
+    - $ref: './common/headers.yaml#/components/parameters/authorizationWriteScopeHeader'
+    - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
+
+  security:
+    - User-Restricted:
+        - write:self-assessment
+
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: './schemas/foreign_property_period_summary_create/request.json'
+        examples:
+          full-expenditure-request:
+            description: Full Expenditure request
+            value:
+              $ref: './examples/foreign_property_period_summary_create/request.json'
+          consolidated-expenditure-request:
+            description: Consolidated Expenditure request
+            value:
+              $ref: './examples/foreign_property_period_summary_create/consolidated_request.json'
+
+  responses:
+    "201":
+      description: Success
```

Comment:
Not sure whats our standard.
Maybe `Success (Created)` or `Created`?

Reply:
same (not fixing-up the RAML for now)